src/tutorials/producers_tutorial.md

```diff
@@ -0,0 +1,1100 @@
+2. Right click **Terminal** and select **Create** followed by **Create New Script**.
+3. Name the script `OutputBuffs`.
+4. Search **Project Content** for `PlaceableOutputBuffData` script.
+5. Drag and drop the **PlaceableOutputBuffData** script into the Hierarchy so it's a child of **OutputBuffs**.
+6. Rename the **PlaceableOutputBuffData** script to `Electricity`.
+7. Select **Electricity** and open the **Properties** window.
+8. Set the **Range** property to `200`.
+
+!!! info "The **Range** property is the radius length of the **Output Buff**."
+
+![!Terminal Placeable Buff](../img/ProducersTutorial/Producer_Terminal_Buff.png){: .center loading="lazy" }
+
+### Set the Properties
+
+The **Terminal Placeable** is not be a Producer Base so the properties need to reflect that.
+
+1. Inside the **Placeables** group, select the **Terminal** script and open the **Properties** window.
+2. Set the **UniqueStorageId** property to `5`.
+3. Set the **Client** property to `Terminal (Client)`.
+4. Set the **Static** property to `Terminal (Static)`.
+5. Set the **PlacementPreview** property to `Terminal (Client)`.
+6. Set the **PlacementOverlapCheckTrigger** property to `Terminal (Trigger)`.
+7. Set the **PlacementAreaTypes** property to `Scifi`.
+8. Deactivate the **IsProducerBase** property.
+
+![!Terminal Placeable Properties](../img/ProducersTutorial/Producer_Terminal_PlaceableProperties.png){: .center loading="lazy" }
+
+## Testing the Placeables
+
+Preview the game and attempt to place the Terminal and Incubator objects on the ground.
+
+The following should work:
+
+- Placeables can only be placed on the Scifi area.
+- Incubator lights up green if within the Terminal radius.
+- Incubator lights up red if not within Terminal radius.
+- Placeables should be prevented from being placed on one another.
+- Players should have collision with placed objects.
+- Placeables should be saved across game plays.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/ProducersTutorial/Producer_TerminalPlaceables.mp4" type="video/mp4" />
+    </video>
+</div>
+
+## Creating an Incubating Egg Producer
+
+The **Incubating Egg Producer** allows an Alien Egg to be placed on the Incubator. It also handles producing drops such as the Alien Raptor and Burnt Egg.
+
+### Examine Producer Properties
+
+#### Producer Phases
+
+A **Producer** uses **phases** to control the produced drops and the visuals.
+
+These are the four phases being used for the **Incubating Egg**:
+
+- **Placed**: When a player places the egg on the incubator it enters this phase.
+- **Building**: If a placed egg is receiving the Electricity Buff, then it begins the building phase.
+- **Ready**: Once the building phase has gone on for a couple seconds, it is now ready to be harvested.
+- **Expired**: If not harvested when ready for a couple more seconds, then the egg expires and becomes burnt.
+
```

Comment:
Maybe switch the above list to a table with the headings "Phase" and "Description"?